app/javascript/store/greetings.js

```diff
@@ -0,0 +1,22 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import axios from 'axios';
+
+const getGreeting = createAsyncThunk('randomGreeting', async () => {
+  const { data } = await axios.get('http://127.0.0.1:3000/api/v1/greetings/random');
```

Comment:
- [ ] Please be careful about the `Axios`  right now not fetching anything, hint( see your dependencies ) and check what it is missing.

Reply:
Hey @dasileker 

Thank you for the feedback. Would you mind sending me the URL giving that result? Also be sure to run `rails db:seed`

Reply:
hello @AdornChoga the URL is the API itself `http://127.0.0.1:3000/api/v1/greetings/random`